docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -346,19 +360,14 @@ For example, the following command runs tests from the specified file and forces
 testcafe ie tests/sample-fixture.js -e
 ```
 
-### -c \<n\>, --concurrency \<n\>
-
-Specifies that tests should run concurrently.
-
-TestCafe opens `n` instances of the same browser and creates a pool of browser instances.
-Tests are run concurrently against this pool, that is, each test is run in the first free instance.
+### -u, --skip-uncaught-errors
 
-See [Concurrent Test Execution](common-concepts/concurrent-test-execution.md) for more information about concurrent test execution.
+When an uncaught error or unhandled promise rejection occurs on the server during test execution, TestCafe stops the test and posts an error message to a report. To ignore these errors, use the `-u`(`--skip-uncaught-errors`) option.
```

Comment:
TestCafe fails this test, not just stops it. Also the situation is possible when some tests run in parallel (i.e.concurrency mode). In this case testcafe fails all of these tests, because we cannot determine which of the test leads to error.

Reply:
I think "stop" is ok. We have the same description for --skip-js-errors
As about note about concurrency, I'll add it.